packages/ui-components/src/components/toggle-tip/toggle-tip.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../assets/styles/globals';
+
+:host {
+	/**
+	 * @prop --button-icon-width: Button's icon width.
+	 * @prop --button-icon-height: Button's icon height.
+	 */
+
+	--button-icon-width: 16px;
+	--button-icon-height: 16px;
+}
+
+kv-icon {
+	--icon-width: var(--button-icon-width);
+	--icon-height: var(--button-icon-width);
+	cursor: pointer;
+}
+
+.toggle-tip-content {
+	width: fit-content;
+	height: fit-content;
+}
+
+.toggle-tip-container {
+	@include kv-font-span-semibold();
+	height: fit-content;
+	width: fit-content;
+	max-width: 240px;
+	word-wrap: break-word;
+	color: kv-color('neutral-0');
+	background-color: kv-color('neutral-6');
+	box-shadow: 0 8px 16px -4px kv-color('neutral-6', 'base', 0.25), 0 0 1px 0 kv-color('neutral-3', 'base', 0.31);
+	user-select: none;
+	border-radius: 2px;
+	padding: $spacing-3x;
+	position: fixed;
+	display: none;
+	z-index: 10000;
```

Comment:
@miguelpinto98  was just a test to check if it could fix the position under the cards but i believe there is another solution for that problem. The actual problem relates to the size of the div where the icon to open the toggle is placed. If the div is too small, we will have problems with the toggle and with the tooltip component. Therefore i will remove this z-index since it was not the solution for the problem.

Reply:
@miguelpinto98 replaced by z-index: 1000; since in the case of the charts from the highcharts, the toggle tip was being drawn behind the chart.

Reply:
suggestion: define a css prop for it, in our use case with highcharts you'll need this higher z-index, however other library users may not need such a high z-index.